### DISCUSSION TITLE
is there any equivalent function in libvips? (cv::copyMakeBorder())

### DISCUSSION THREAD

**Discussion by oldercoder** [libvips/libvips#3683] [Q&A] [answered]
Opencv has a "cv::copyMakeBorder()" function to make new image and extend edge pixel like constant/reflect/replicate etc., I'm be curious if libvips does same.

**Comment by jcupitt** [comment, marked as answer]
Hi @oldercoder,

You need `embed`:

https://www.libvips.org/API/current/libvips-conversion.html#vips-embed

**Reply by oldercoder** [reply]
Thank you @jcupitt , I'll have a try.
